contrib/nydus-snapshotter/cmd/containerd-nydus-grpc/pkg/command/flags.go

```diff
@@ -207,6 +214,7 @@ func Validate(args *Args, cfg *config.Config) error {
 	cfg.MetricsFile = args.MetricsFile
 	cfg.EnableStargz = args.EnableStargz
 	cfg.DisableCacheManager = args.DisableCacheManager
+	cfg.EnableKata = args.EnableKata
```

Comment:
Seems not a generic option name as `EnableKata`.
Let's try to think of a more generic one?